### DISCUSSION TITLE
Container keeps failing when deploying with "Apps" on digital ocean

### DISCUSSION THREAD

**Discussion by AchrafBardan** [serversideup/docker-php#266] [Q&A]
[ibtikar-api-deploy.txt](https://github.com/serversideup/docker-php/files/13798375/ibtikar-api-deploy.txt)
[ibtikar-api-build.txt](https://github.com/serversideup/docker-php/files/13798376/ibtikar-api-build.txt)

When deploying it keeps returning:
`s6-notifyoncheck: fatal: unable to sanity-check current service directory: Invalid argument`

Searched the entire internet for this and found nothing. It works local.

**Comment by jaydrogers** [comment]
You might want to consider trying our `unit` images in the beta.

`fpm-nginx` requires S6 Overlay, which isn't compatible with all hosts https://release-v3-0.docker-php.pages.dev/docs/guide/using-s6-overlay#this-disadvantage-of-s6-overlay

**Comment by maximilliangeorge** [comment]
> [ibtikar-api-deploy.txt](https://github.com/serversideup/docker-php/files/13798375/ibtikar-api-deploy.txt) [ibtikar-api-build.txt](https://github.com/serversideup/docker-php/files/13798376/ibtikar-api-build.txt)
> 
> When deploying it keeps returning: `s6-notifyoncheck: fatal: unable to sanity-check current service directory: Invalid argument`
> 
> Searched the entire internet for this and found nothing. It works local.

Hey @AchrafBardan, did you manage to get it working? I'm the exact same situation. Trying to deploy to DigitalOcean App Platform.

**Reply by AchrafBardan** [reply]
Hey @maximilliangeorge 
Dis not manage to fix it so i just went with laravel forge. The way to fix this is by not using s6 overlay becausemost of these app platforms like digital ocean are not one to one linux distros.
